### PR TITLE
make MoCO model not distributed across multiple GPUs

### PR DESCRIPTION
**Pull request by nickgg**
The moco model is defined as a distrubuted NN, but we fix it to a single GPU in the benchmarks. Removing all the cross GPU communication for simplicitys sake.

### CODE REVIEW

**Comment by wconstab** [comment]
oh, can you also fix default device?  And check off the box here: https://github.com/pytorch/benchmark/projects/1

**Comment by wconstab** [comment]
and make sure to raise NotImplemeted for jit?  It looks like jit is silently disabled for now.

**Comment by nickgg** [comment]
I could comment out the code rather than delete it, but if/when we want to do distributed benchmarks probably easier to grab the copy from the original repo anyway.

The calls to nn.dist (eg the gathers) won't work if its not a DDP.

What do you mean by fix the default device and check off the project?

**Comment by wconstab** [comment]
ok, deleting or commenting is fine just wanted to ask.

oops, i sent the wrong link.  check the moco specific 'nit' in this issue https://github.com/pytorch/benchmark/issues/65

File: models/moco/hubconf.py [hunk 1]
```diff
@@ -127,9 +121,6 @@ def train(self, niterations=1):
 
         Leave warmup to the caller (e.g. don't do it inside)        
         """
-        if self.device != "cuda":
```
Comment (wconstab) [inline models/moco/hubconf.py:130]: i'm confused - above you raise on CPU, but here you allow it?

Comment (nickgg) [reply models/moco/hubconf.py:130]: sorry I'm trying to do this quickly in between some other work, this is a typo.

Comment (wconstab) [reply models/moco/hubconf.py:130]: hah you and me both. i know the feeling..

**Comment by xuzhao9** [comment]
Addressed, closing this PR.

**Pull request closed without merging**